src/modules/qt/CMakeLists.txt

```diff
@@ -45,6 +45,14 @@ function(mlt_add_qt_module ARG_TARGET)
       ${YML}
     )
 
+    # Note: We explicitly define a common macro and file name here because this
+    # function creates two separate targets (mltqt and mltqt6) that share the
+    # same source code. This ensures a single, consistent export macro can be
+    # used throughout the C++ files.
```

Comment:
But this will fail if MLT will be built against both Qt versions or do I miss something?